Food Order Website/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Foodie</title>
+    <link rel="stylesheet" href="styles.css">
+
+    <nav class="navbar">
+        <ul>
+            <li><a href="index.html">HOME</a></li>
```

Comment:
this should link to /index.html for ease of naivation back to repo home page. The you need to add your project to the LandingPages/index.html so it can be viewed from te home page